rakefile

```diff
@@ -0,0 +1,89 @@
+require 'rototiller'
+
+task :default => :acceptance
+
+desc 'Run Performance setup'
+rototiller_task :performance do |t|
+  t.add_env({:name => 'ENVIRONMENT_TYPE', :message => 'Either gatling or clamps', :default => 'gatling'})
+
+  if ENV['ENVIRONMENT_TYPE'] == 'clamps'
+    default_tests = 'setup/install'
+  else
+    default_tests = 'setup/install/00_pre_install/,setup/install/10_install/00_install_pe.rb,setup/install/30_post_install/'
+  end
+
+  # env vars needed for Beaker
+  t.add_command do |command|
+    command.name = "bundle exec beaker"
+    command.add_env({:name => 'BEAKER_EXECUTABLE'})
+
+    command.add_option do |option|
+      option.name = '--config'
+      option.message = 'The configuration file that Beaker will use'
+      option.add_argument do |arg|
+        arg.add_env({:name => 'CONFIG'})
+      end
+    end
+    command.add_option do |option|
+      option.name = '--keyfile'
+      option.message = 'The SSH key used to access a SUT'
+      option.add_argument do |arg|
+        arg.name = "#{ENV['HOME']}/.ssh/id_rsa-acceptance"
+        arg.add_env({:name => 'BEAKER_KEYFILE'})
+      end
+    end
+    command.add_option do |option|
+      option.name = '--log-level'
+      option.message = 'The log level under which you want beaker to run'
+      option.add_argument do |arg|
+        arg.name = 'debug'
+        arg.add_env({:name => 'BEAKER_LOG_LEVEL'})
+      end
+    end
+    command.add_option do |option|
+      option.name = '--pre-suite'
+      option.message = 'Beaker pre-suite'
+      option.add_argument do |arg|
+        arg.name = default_tests
+        arg.add_env({:name => 'BEAKER_PRE_SUITE'})
+      end
+    end unless ENV['BEAKER_PRE_SUITE'] == ''
+    command.add_option do |option|
+      option.name = '--helper'
+      option.message = 'Setup helper required for installing PE'
+      option.add_argument do |arg|
+        arg.name = 'setup/helpers/classification_helper.rb,setup/helpers/ldap_helper.rb'
+        arg.add_env({:name => 'HELPER_PATH'})
```

Comment:
Is this a standard names used in lots of other places?  Otherwise it doesn't seem to line up with the values, which are actually a list of helpers to load.  Plus it doesn't have "BEAKER" in front like the previous names